testsuite/python/correlation.py

```diff
@@ -50,13 +50,13 @@ def test_square_distance_componentwise(self):
         s = self.system
         s.part.add(id=0, pos=(0, 0, 0), v=(1, 2, 3))
 
-        O = espressomd.observables.ParticlePositions(ids=(0,))
-        C2 = espressomd.accumulators.Correlator(
-            obs1=O, tau_lin=10, tau_max=2, delta_N=1,
+        obs = espressomd.observables.ParticlePositions(ids=(0,))
+        acc = espressomd.accumulators.Correlator(
+            obs1=obs, tau_lin=10, tau_max=10.0, delta_N=1,
```

Comment:
I accidentally reverted the `tau_max` value during the merge. However by setting `tau_max=2` in 7864a096 you moved from the multiple tau correlator to the linear tau correlator (since  `tau_lin < tau_max`), which according to the docs should be slower in this case because `tau_max = 200 * dt`. Was this change intentional?

Reply:
The merge was fixed.